Sources/TuistGenerator/Linter/TargetLinter.swift

```diff
@@ -292,6 +293,20 @@ class TargetLinter: TargetLinting {
         }
         return []
     }
+
+    private func lintOnDemandResourcesTags(target: Target) -> [LintingIssue] {
+        guard let odrTags = target.onDemandResourcesTags else { return [] }
```

Comment:
Lets name this one `onDemandResourcesTags` instead to prevent confusion further down from a not very explicit name.